apps/qa/bash/docker_build_and_publish.sh

```diff
@@ -9,6 +9,12 @@ set_error_traps
 
 cp $ROOT_DIR/pyproject.toml $PROJECT_DIR
 cp $ROOT_DIR/python/constraints.txt $PROJECT_DIR
+
+mkdir $PROJECT_DIR/products
+for dataset in "cbbr" "colp" "cpdb" "developments" "facilities" "pluto" "zoningtaxlots"; do
```

Comment:
Per @sf-dcp 's question, this is a bit hacky. I'll add a comment in the code explaining why it's currently like this

Reply:
another way that comes to mind is moving the product recipe.yml files into dcpy, but that doesn't seem worth the trouble yet